Userland/Services/WindowServer/main.cpp

```diff
@@ -44,7 +44,7 @@ ErrorOr<int> serenity_main(Main::Arguments)
     auto wm_config = TRY(Core::ConfigFile::open("/etc/WindowServer.ini"));
     auto theme_name = wm_config->read_entry("Theme", "Name", "Default");
 
-    auto theme = Gfx::load_system_theme(DeprecatedString::formatted("/res/themes/{}.ini", theme_name));
+    auto theme = TRY(Gfx::load_system_theme(DeprecatedString::formatted("/res/themes/{}.ini", theme_name)));
     VERIFY(theme.is_valid());
```

Comment:
Since we used to `VERIFY` that this passes and this is in the start up of the window server I think this is a good spot for MUST.

Reply:
Oh if there's a nice error then this is fine. I defintely don't know much about WindowServer startup

Reply:
TRY produces this on the screen when an error is raised

![Screenshot_20221207_111801](https://user-images.githubusercontent.com/99915288/206166006-b2396478-74f8-4fb5-8c4b-27425e0b43f9.png)

MUST results in

![Screenshot_20221207_111959](https://user-images.githubusercontent.com/99915288/206166143-ac804b21-9f5f-49a1-b43e-873b3ba983cb.png)

with no further info in the debug console so I think I prefer the TRY method.

Additionally, the is_valid() just checks for an array existing rather than a valid theme so I think that it is possibly redundant here now that we propagate errors up and would be inclined to remove it.

Reply:
Yes I don't believe it can be hit anymore as the AnonymousBuffer created further down the chain is also wrapped with TRY and would fail much sooner if that failed to allocate. I will be removing the VERIFY